test/org/sagebionetworks/bridge/validators/StudyValidatorTest.java

```diff
@@ -160,15 +160,57 @@ public void acceptsMultipleValidTechnicalEmailAddresses() {
     @Test
     public void rejectsInvalidTechnicalEmailAddresses() {
         study.setTechnicalEmail("test@test.com,asdf,test2@test.com");
-        assertValidatorMessage(INSTANCE, study, "technicalEmail", "'asdf' is not a valid email address");
+        assertValidatorMessage(INSTANCE, study, "technicalEmail", StudyValidator.EMAIL_ERROR);
     }
     
     @Test
-    public void requiresMissingTechnicalEmail() {
```

Comment:
Instead of deleting requiresMissingTechnicalEmail(), should we replace it with a test that a study without a technical email is still valid?